express/code/blocks/logo-row/logo-row.css

```diff
@@ -24,8 +28,9 @@ main .logo-row .block-row {
 
 main .logo-row .text-column {
     margin: auto auto 16px auto;
-    max-width: 300px;
     text-align: center;
+    padding-left: 16px;
```

Comment:
No sure if this applies well for Milo: do we prefer unit measures over REM?, and for this values, could we do variables instead?

Reply:
good question... I don't think we have a finalized standard now and our design system has been constantly evolving still. So either way is usually fine. But for color value and font sizes we do prefer variables.